generators/spa/templates/Vue3JS/common/packages/MSBuildTasks.1.4.0.88/tools/MSBuild.psm1

```diff
@@ -0,0 +1,146 @@
+function Resolve-ProjectName {
```

Comment:
@alejoroman0605 Why isn't this using the same build files as the other module templates?